test/EcdsaXi2023Cryptosuite.spec.js

```diff
@@ -0,0 +1,372 @@
+/*!
+ * Copyright (c) 2024 Digital Bazaar, Inc. All rights reserved.
+ */
+import {expect} from 'chai';
+
+import jsigs from 'jsonld-signatures';
+const {purposes: {AssertionProofPurpose}} = jsigs;
+
+import * as EcdsaMultikey from '@digitalbazaar/ecdsa-multikey';
+import {
+  credential,
+  ecdsaMultikeyKeyPair,
+  ecdsaSecp256KeyPair
+} from './mock-data.js';
+import {createCryptosuite} from '../lib/index.js';
+import {DataIntegrityProof} from '@digitalbazaar/data-integrity';
+
+import {loader} from './documentLoader.js';
+
+const documentLoader = loader.build();
+const extraInformation = '6d721ae5d334cead832a8576bdd24d9a';
```

Comment:
This should be a `Uint8Array` instead of a string. We shouldn't impose any particular string-encoding of the extra information (like `hex`) -- and I'm thinking the information shouldn't even be hashed yet, but the 
cryptosuite should do that so there's a guarantee on the crypto-primitives being used and so on.

Reply:
Fixed